src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RandomNumberGenerator.cs

```diff
@@ -146,6 +146,26 @@ public static int GetInt32(int toExclusive)
             return GetInt32(0, toExclusive);
         }
 
+        /// <summary>
+        /// Creates an array of bytes with a cryptographically strong random sequence of values.
+        /// </summary>
+        /// <param name="count">The number of bytes of random values to create.</param>
+        /// <returns>
+        /// An array populated with cryptographically strong random values.
+        /// </returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// The <paramref name="count" /> parameter is less than zero.
```

Comment:
Based on the ArgumentOutOfRangeException example at https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions, I think this should be

```suggestion
        /// <paramref name="count" /> is less than zero.
```